Assets/Scripts/Game/UserInterfaceWindows/DaggerfallInventoryWindow.cs

```diff
@@ -1241,37 +1255,45 @@ protected void NextVariant(DaggerfallUnityItem item)
                 Refresh(false);
         }
 
-        protected bool CanCarry(DaggerfallUnityItem item)
+        protected int CanCarryAmount(DaggerfallUnityItem item)
         {
             // Check weight limit
-            if (GetCarriedWeight() + (item.weightInKg * item.stackCount) > playerEntity.MaxEncumbrance &&
-                item.ItemGroup != ItemGroups.Transportation && item.TemplateIndex != (int)Weapons.Arrow)
+            int canCarry = item.stackCount;
+            //  item.TemplateIndex != (int)Weapons.Arrow must have been there for lack of stack splitting
```

Comment:
No I don't think so. Stack splitting should not require changes to encumbrance code.